embedded-graphics/src/draw_target.rs

```diff
@@ -0,0 +1,322 @@
+use crate::{
+    drawable::{self, Drawable},
+    geometry::{Point, Size},
+    pixelcolor::PixelColor,
+    primitives::{self, Primitive},
+    style::{PrimitiveStyle, Styled},
+};
+
+/// Defines a display that can be used to render [`Drawable`] objects.
+///
+/// To to add embedded-graphics support to a display driver, `DrawTarget` must be implemented. Once
+/// a `DrawTarget` is defined, it can be used to render [`Drawable`]s. Note that any iterator over
+/// [`Pixel`]s can be drawn as [`Drawable`] is implemented for `Iterator<Item = Pixel<C:
+/// PixelColor>>`. See the [`Drawable`] trait documentation for more details.
+///
+/// `DrawTarget` also provides a way to define accelerated drawing methods for graphical primitives
```

Comment:
`provides a way` isn't very informative. I would mention that this can be done by overriding the default implementations with hardware accelerated ones.